lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -0,0 +1,246 @@
+                       priority=priority, goto_table=self.next_table)
+
+        # Add ARP flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                               in_port=self.config.mtr_port,
+                               arp_tpa=ue_ip_adr)
+        if sid:
+            actions = [parser.OFPActionSetField(metadata=sid)]
+
+        flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
+                       priority=priority, goto_table=self.next_table)
+
+
+    def _delete_tunnel_flows(self, i_teid:int, ue_ip_adr:str):
+
+        # Delete flow for gtp port
+        match = MagmaMatch(tunnel_id=i_teid, in_port=self.config.gtp_port)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete flow for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self._uplink_port, ipv4_dst=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self.config.mtr_port,ipv4_dst=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete ARP flow for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                           in_port=self._uplink_port, arp_tpa=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete ARP flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                           in_port=self.config.mtr_port, arp_tpa=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+
+    def _add_discard_data_tunnel_flows(self, precedence:int, i_teid:int,
+                                            ue_ip_adr:str):
+        priority = self.convert_precedence_to_priority(precedence)
+        # discard uplink Tunnel
+        match = MagmaMatch(tunnel_id=i_teid, in_port=self.config.gtp_port)
+        cookie = self._cookie
+
+        flows.add_flow(self._datapath, self.tbl_num, match,
+                       priority=priority + 1,cookie=cookie)
+
+        # discard downlink Tunnel for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self._uplink_port, ipv4_dst=ue_ip_adr)
+        cookie = self._cookie + 1
+
+        flows.add_flow(self._datapath, self.tbl_num, match,
+                       priority=priority + 1,cookie=cookie)
+
+        # discard downlink Tunnel for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self.config.mtr_port,ipv4_dst=ue_ip_adr)
+        cookie = self._cookie + 1
+ 
+        flows.add_flow(self._datapath, self.tbl_num, match,
+                       priority=priority + 1,cookie=cookie)
+
+    def _add_forward_data_tunnel_flows(self, precedence:int, i_teid:int,
```

Comment:
same comment here, this is mainly used for CSFB where the discard rules are removed to resume active or idle mode flow rules. We probably don't need this.